mysql scripts/appointmentDB.sql

```diff
@@ -173,15 +182,20 @@ INSERT INTO Location (title, address)
 	VALUES ("Yellow", "591 Oak Avenue");
 
 -- sample appointment with answers
-
 INSERT INTO Appointment (scheduledTime, locationId) 
 	VALUES ("2017-05-28 16:30:00", 1);
 	
 INSERT INTO Answer (string, questionId) 
-	VALUES ("Ralph Schmidt", 
+	VALUES ("Ralph", 
 	(SELECT questionId 
 		FROM Question 
-		WHERE string="Name"));
+		WHERE string="First Name"));
+        
+INSERT INTO Answer (string, questionId)
+	VALUES ("Schmidt",
+    (SELECT questionId
```

Comment:
There are actually several places in this file where there are mixed spaces and tabs.